src/UI/Component/Symbol/Factory.php

```diff
@@ -15,17 +15,17 @@ interface Factory
 	 * ---
 	 * description:
 	 *   purpose: >
-	 *     Icons are quickly comprehensible and recognizable graphics.
-	 *     They indicate the functionality or nature of a text-element or context:
+	 *     Icons are quickly comprehensible and recognizable graphics that MUST be used with a text label.
```

Comment:
Do not state rules in the descriptions section. Please remove. If you are sure this rule is needed, move to the rules/usage section. Note that an additional rule would need a JF decision, so better remove in this PR.